### PR TITLE
Typos

### PR DESCRIPTION
**Pull request by luzpaz**
found via `codespell`

### CODE REVIEW

File: master/buildbot/test/unit/test_schedulers_forcesched.py [hunk 1]
```diff
@@ -390,7 +390,7 @@ def do_ParameterTest(self,
                     import xerox
                     formated = self.formatJsonForTest(gotJson)
                     print(
-                        "You may update the test with (copied to clipboard):\n" + formated)
+                        "You may update the test with (copied to clipboard):\n" + formatted)
```
Comment (sa2ajj) [inline master/buildbot/test/unit/test_schedulers_forcesched.py]: please see above the actual variable name

Comment (luzpaz) [reply master/buildbot/test/unit/test_schedulers_forcesched.py]: @sa2ajj nice catch. sorry to miss it. Any possibility you'd consider changing the variable name to the correct spelling ?

Comment (tardyp) [reply master/buildbot/test/unit/test_schedulers_forcesched.py]: @luzpaz of course, please fix the variable name

Comment (luzpaz) [reply master/buildbot/test/unit/test_schedulers_forcesched.py]: done. re-pushed with requested revision.

File: master/buildbot/test/unit/test_schedulers_forcesched.py [hunk 2]
```diff
@@ -388,9 +388,9 @@ def do_ParameterTest(self,
             if gotSpec != expectSpec:
                 try:
                     import xerox
-                    formated = self.formatJsonForTest(gotJson)
+                    formatted = self.formatJsonForTest(gotJson)
                     print(
-                        "You may update the test with (copied to clipboard):\n" + formated)
+                        "You may update the test with (copied to clipboard):\n" + formatted)
                     xerox.copy(formated)
```
Comment (tardyp) [inline master/buildbot/test/unit/test_schedulers_forcesched.py]: formated here also (see the pylint and flake8 errors)

Comment (luzpaz) [reply master/buildbot/test/unit/test_schedulers_forcesched.py]: @tardyp thanks, can you share what I need to run to check pylint and flake8?  any specific flags?

Comment (tardyp) [reply master/buildbot/test/unit/test_schedulers_forcesched.py]: you can just run ```make virtualenv``` and then ```make flake8``` and ```make pylint```

Or you can push the code, and wait for our CI to tell you the results:

image

**Pull request merged**